src/components/shared/SearchInput.js

```diff
@@ -38,6 +38,11 @@ class SearchInput extends Component {
 
   static defaultProps: Object;
 
+  componentDidMount() {
+    this.input.focus();
+    this.input.select();
```

Comment:
We were using it but we were making a selection of the end to the end? So I wasn't sure what exactly that was achieving. I don't think it nor `.select()` make a ton of sense so I don't care for keeping them.